docs/docs/features/studio.md

```diff
@@ -10,6 +10,34 @@ ZMK Studio is in beta. Although every effort has been made to provide a stable e
 
 ZMK Studio provides runtime update functionality to ZMK powered devices, allowing users to change their keymap layers without flashing new firmware to their keyboards. Studio is still under active development, and is not yet ready for casual end user use.
 
+## Capabilities
+
+ZMK Studio currently has the following capabilities:
+
+| Legend: | ✅ Supported | 🚧 Under Development | 💡 Planned | ❌ Not Planned | ? Pete, help pls |
+| :------ | :----------- | :------------------- | :--------- | :------------- | ---------------- |
+
+| Studio Status | **Feature or Capability**                                                                         |
+| :-----------: | ------------------------------------------------------------------------------------------------- |
+|      ✅       | Making keymap changes while keyboard is in use                                                    |
+|      ✅       | Making changes while connected via USB                                                            |
+|      ✅       | Native app for: Windows, Linux, MacOS                                                             |
+|      ✅       | Making changes while connected via BLE (Linux web-app & native apps only)                         |
+|      ✅       | Assigning [predefined behaviors](../keymaps/behaviors/index.mdx) to keys                          |
+|      ✅       | Assigning [user-defined behaviors](../keymaps/behaviors/index.mdx#user-defined-behaviors) to keys |
+|      💡       | Defining new [user-defined behaviors](../keymaps/behaviors/index.mdx#user-defined-behaviors)      |
+|       ?       | [Configuring predefined behaviors](../config/behaviors.md)                                        |
+|      💡       | Defining [combos](../keymaps/combos.md)                                                           |
+|      💡       | Defining [conditional layers](../keymaps/conditional-layers.md)                                   |
+|       ?       | Assigning behaviors to encoders                                                                   |
```

Comment:
I am not really interested in this *yet* since we're still up in the air on moving encoders to the input API, etc. so this is pretty low on my list at this time.

Reply:
What I've done is added another category "low priority" and put this point under it.